bin/cucumber-js

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require('../lib/cli/run.js').default();
+require('../lib/cli/run.js').default()
```

Comment:
What's the reason for this change?

Reply:
Ah, I think this was just an artifact of me initially making some changes here and then backing them out. Reverted now for clarity.